packages/shared/src/components/modals/MarketingCTAModal.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from 'react';
+import { Modal, ModalProps } from './common/Modal';
+import { ButtonSize } from '../buttons/Button';
+import type { MarketingCTA } from '../cards';
+import { CTAButton, Description, Header, Title } from '../cards';
+import { CardCover } from '../cards/common/CardCover';
+
+export interface NewRankModalProps extends ModalProps {
+  marketingCTA: MarketingCTA;
+}
+export const MarketingCTAModal = ({
+  marketingCTA,
+  onRequestClose,
+  ...modalProps
+}: NewRankModalProps): ReactElement => {
+  const { tagColor, tagText, title, description, image, ctaUrl, ctaText } =
+    marketingCTA;
+
+  const onModalClose: typeof onRequestClose = (param) => {
+    onRequestClose(param);
+  };
+
+  return (
+    <Modal
+      {...modalProps}
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.Small}
+      onRequestClose={onModalClose}
+    >
+      <div className="p-6 !pt-4">
```

Comment:
Why the ! here?

Reply:
Probably ordering of classes?

Reply:
Yeah basically want 6 all round, but top 4.
This was easier then doing 4 loose ones, but let me know if you don't agree 👍

Reply:
Nah it's fine